src/pages/dashboard/index.jsx

```diff
@@ -1,67 +1,162 @@
+  const handleClickOutside = (event) => {
+    if (
+      !event.target.closest('.search-cohorts-results') &&
+      !event.target.closest('#input-wrapper-search-bar')
+    ) {
+      setIsSearchResVisible(false)
+      setIsStudentModalVisible(false)
+      setSelectedProfileId(null)
+    }
+  }
 
-        // Open the modal!
-        openModal()
+  useEffect(() => {
+    document.addEventListener('click', handleClickOutside)
+    return () => {
+      document.removeEventListener('click', handleClickOutside)
     }
+  }, [])
+
+  const onClickStudent = (id) => {
+    setSelectedProfileId(id)
+    setIsStudentModalVisible(true)
+  }
+
+  const { openModal, setModal } = useModal()
+
+  const showModal = () => {
+    setModal('Create a post', <CreatePostModal />)
+    openModal()
+  }
+
+  return (
+    <>
+      <main>
+        <Card name='create-post'>
+          <div className="create-post-input">
+            <div className="profile-icon">
+              <p>AJ</p>
+            </div>
+            <Button text="What's on your mind?" onClick={showModal} />
+          </div>
+        </Card>
+
+        <Posts />
+      </main>
+
+      <aside>
+        <Card>
+          <form onSubmit={(e) => e.preventDefault()} onClick={onClickSearchBar}>
+            <div id="input-wrapper-search-bar">
+              <SearchIcon />
+
+              <input
+                type="search"
+                name="Search"
+                value={searchVal}
+                onChange={onChange}
+                placeholder="Search for people"
+              />
+            </div>
+          </form>
+        </Card>
+
+        {isSearchResVisible && result.length === 0 && (
+          <article className="search-cohorts-results">
+            <p>People</p>
+
+            <div className="divider-search-bar"></div>
+
+            <p>Sorry, no results found.</p>
+
+            <p>Try changing your search term.</p>
+
+            <button>Edit search</button>
+          </article>
+        )}
+
+        {isSearchResVisible && result.length >= 1 && (
+          <article className="search-cohorts-results">
+            <p>People</p>
+
+            <div className="divider-search-bar"></div>
+
+            <ul>
+              {result.map((cohort) => (
+                <li className="user-search-card" key={cohort.id}>
+                  <ProfileCircle
+                    initials={[cohort.firstName[0], cohort.lastName[0]]}
+                    hasCascadingMenu={false}
+                  />
+
+                  {isStudentModalVisible && selectedProfileId === cohort.id && (
+                    <Menu className="profile-circle-menu" ref={menuRef}>
+                      <MenuItem icon={<ProfileIcon />} text="Profile" />
+                    </Menu>
+                  )}
+
+                  <div>
+                    <b>{`${cohort.firstName} ${cohort.lastName}`}</b>
```

Comment:
let's just use a span and css - `b` tag isn't really used anymore

Reply:
Ok, no problems! Just wondering, why is that?